libraries/scripting/dependencies-maven-all/build.gradle.kts

```diff
@@ -0,0 +1,89 @@
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+
+description = "Shaded Maven dependencies resolver"
+
+val JDK_18: String by rootProject.extra
+val jarBaseName = property("archivesBaseName") as String
+
+val embedded by configurations
+
+embedded.apply {
+    exclude("org.slf4j", "slf4j-api")
+    exclude("org.eclipse.aether", "aether-api")
+    exclude("org.eclipse.aether", "aether-util")
+    exclude("org.eclipse.aether", "aether-spi")
+}
+
+plugins {
+    kotlin("jvm")
+    id("jps-compatible")
+}
+
+dependencies {
+    embedded(project(":kotlin-scripting-dependencies-maven")) { isTransitive = false }
+    embedded(project(":kotlin-scripting-dependencies")) { isTransitive = false }
+
+    embedded("org.eclipse.aether:aether-connector-basic:1.1.0")
+    embedded("org.eclipse.aether:aether-transport-wagon:1.1.0")
+    embedded("org.eclipse.aether:aether-transport-file:1.1.0")
+    embedded("org.apache.maven:maven-core:3.8.1")
+    embedded("org.apache.maven.wagon:wagon-http:3.4.3")
```

Comment:
Please doublecheck the licenses, transitively as well, we are not allowed to pack anything GPL into our files.

Reply:
Is EPL 1.0 OK?

Reply:
I've checked, there are three licences in dependencies of this module and their transitive dependencies:
1. Apache 2.0
2. EPL 1.0
3. MIT
I believe they all are sufficient ones.

Reply:
Will need to check about the EPL on Monday.

Reply:
please copy used libraries' licences to the license/third_party